### PR TITLE
Update index page (due 12/05/22)

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Update index page 
- Update u-heading classes

## QA

- Go to https://juju-is-397.demos.haus 
- Please compare page against [copy doc](https://docs.google.com/document/d/147spDkSVH0YaEGKZ4tDi7i28zzhIcacF1hBOMHvxaC0/edit#)

## Issue / Card

Fixes [#5269](https://github.com/canonical-web-and-design/web-squad/issues/5269)

### CODE REVIEW

File: templates/index.html [hunk 1]
```diff
@@ -8,35 +8,37 @@
...
-
+        <p>Juju is the Charmed Operator Framework; an open source framework that uses <a href="/docs/sdk/charmed-operators">Charmed Operators</a>, or 'charms’, to deploy cloud infrastructure and applications and manage their operations from Day 0 through Day 2. With Juju, you can install, maintain, upgrade, and integrate applications across Kubernetes clusters, containers, virtual machines, and bare metal machines, on public or private clouds. <a href="/docs/olm/installing-juju">Take control of your hybrid environment</a>.</p>
         <div class="row">
-          <div class="col-6">
-            <p>Juju is a Charmed Operator Framework, composed of a Charmed Operator Lifecycle Manager and the Charmed Operator SDK. Deploy, <strong>integrate</strong>, and manage Kubernetes, container and VM-native applications seamlessly across hybrid clouds. Juju drives Day 0 through Day 2 operations in your complex environment.</p>
+          <div class="col-2">
+            <h5 class="u-no-margin--bottom">Learn</h5>
+            <p>Learn how to manage charms with the <a href="/about" class="u-align-text--left">Charmed Operator Lifecycle Manager (OLM) docs</a>.</p>
```
Comment (mtruj013) [inline templates/index.html:17]:
```suggestion
            <p>Learn how to manage charms with the <a href="/docs/olm" class="u-align-text--left">Charmed Operator Lifecycle Manager (OLM) docs</a>.</p>
```

**Review by mtruj013 — APPROVED** [review]
LGTM, just one small change:

**Comment by carkod** [comment]
@minkyngkm @sowasred2012 shouldn't we do [this one too](https://github.com/canonical-web-and-design/web-squad/issues/5274) in this PR? if its just only typos?

**Comment by tmihoc** [comment]
Small typo: In the "What is Juju?" section, "Minimize" should be "Minimise".

**Comment by minkyngkm** [comment]
Thanks @tmihoc! It has been fixed.

**Comment by jameinel** [comment]
Trying to see the new layout on a "reasonable size page" this was the old page design:
image

And you can see that it gets the full "When you need to ... Juju helps you to take control" as fitting above the fold.
With the same size window the new site loses that:
image


We also lost the visual appeal of the 2 boxes above "Install the Operator Lifecycle Manager". I think it would be reasonable to do 3 boxes with a Learn/Try/Build. Maybe we can find a wording that still retains the prettiness of the old design?

**Comment by sowasred2012** [comment]
@jameinel adding the boxes pushes the content below even further down, and doesn't look too great unless they're split onto two rows - I think this needs more thinking about, and design time:

3 column layout:
image

2 + 1 layout:
image

**Comment by sowasred2012** [comment]
@tmihoc - thanks, I tried that suggestion out, let me know what you think:

With boxes:
Image Pasted at 2022-5-13 16-30

Without boxes:
Image Pasted at 2022-5-13 16-30-2

**Comment by tmihoc** [comment]
Neither option seems to help us solve the problem of the interrupted "When..control" block. This said, I personally like the second version best of all the ones I've seen (shortened text, no boxes). @jameinel ?

**Comment by tmihoc** [comment]
> Neither option seems to help us solve the problem of the interrupted "When..control" block. This said, I personally like the second version best of all the ones I've seen (shortened text, no boxes). @jameinel ?

In the second version: How about we replace

> **Build** a charm with the [Charmed Operator Software Development Kit (SDK)](https://juju.is/docs/sdk)

with the shorter

> > **Build** a charm with the [Charmed Operator SDK](https://juju.is/docs/sdk)

? Would that fix the interrupted "When...control" block problem?

**Comment by jameinel** [comment]
The short text without blocks does, indeed, look the best of these options.
John
=:->

**Comment by tmihoc** [comment]
Thanks @minkyngkm and @sowasred2012 for helping get it live! I was wondering if we could make one last attempt at making the "When...control." fit on the first screen. How about we make the font for the "When you need to......Juju helps you take control" bit a little smaller?

**Pull request merged**